SignalServiceKit/src/Messages/OWSMessageDecrypter.m

```diff
@@ -682,6 +684,8 @@ - (void)notifyUserForErrorMessage:(TSErrorMessage *)errorMessage
 {
     NSString *hexEncodedPublicKey = [LKDatabaseUtilities getMasterHexEncodedPublicKeyFor:envelope.source in:transaction] ?: envelope.source;
     TSThread *contactThread = [TSContactThread getOrCreateThreadWithContactId:hexEncodedPublicKey transaction:transaction];
+    contactThread.shouldThreadBeVisible = true;
```

Comment:
Why does the thread need to be visible?

Reply:
When recovering from seed, if we get rid of all the error messages from messages before restoration, the thread won't show up in HomeVC.

Reply:
Okay, but what if an error occurs in a thread that should be invisible?